types/survicate__react-native-survicate/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for @survicate/react-native-survicate 1.0
+// Project: https://github.com/Survicate/react-native-survicate#readme
+// Definitions by: Ian Mobley <https://github.com/iMobs>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/survicate__react-native-survicate/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me